tests/test_client.py

```diff
@@ -402,6 +403,25 @@ def test_request_invalid_http_headers():
     assert str(value_error.value).startswith("cannot override reserved HTTP header")
 
 
+def test_enabling_https(monkeypatch):
```

Comment:
Let's test all of the following forms:

1. `server:443` => https.
2. `https://server:443` => https
3. `http://server:443` => http (since scheme is specified in the URL)

Reply:
added in https://github.com/trinodb/trino-python-client/pull/109/commits/306ac9b675d5283f5d9288fe98b47dc3e808b5e0